common/openapi.yml

```diff
@@ -145,6 +145,25 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorReport'
+  /api/v1/datasets/{id}/export:
+    post:
```

Comment:
Is a post the right call here?

Reply:
I think so, I think it makes more sense than `PUT` for this operation.